chainer/functions/normalization/group_normalization.py

```diff
@@ -50,13 +52,18 @@ def forward(self, inputs):
         self.retain_inputs((0, 1))
         x, gamma, beta = inputs
 
+        interm_dtype = numpy.promote_types(x.dtype, gamma.dtype)
```

Comment:
Could you use this dtype instead of x's dtype in the `xp.sqrt(...)` below (on line 72)? Maybe you could strengthen the tolerances after this in the tests but I'm not sure.

Reply:
Removed the dtype argument instead of using `interm_dtype` because the dtype of `inv_std` is already `interm_dtype` on line 72.